x-pack/plugins/apm/public/components/app/correlations/ml_latency_correlations.tsx

```diff
@@ -229,9 +230,31 @@ export function MlLatencyCorrelations({ onClose }: Props) {
       {
         width: '116px',
         field: 'correlation',
-        name: i18n.translate(
-          'xpack.apm.correlations.latencyCorrelations.correlationsTable.correlationLabel',
-          { defaultMessage: 'Correlation' }
+        name: (
+          <EuiToolTip
+            content={i18n.translate(
+              'xpack.apm.correlations.latencyCorrelations.correlationsTable.correlationColumnDescription',
+              {
+                defaultMessage:
+                  'The impact of a field on the latency of the service, ranging from 0 to 1.',
```

Comment:
I might add one sentence which is something like...
```suggestion
                  'The impact of a field on the latency of the service, ranging from 0 to 1. The larger the value the more the field increases latency.',
```

Reply:
Thanks for the feedback!  This tooltip was already merged into https://github.com/elastic/kibana/pull/99905, so I've amended the text in f6c2155b1d1b94bd81954a631a6b3bb5efa4ca9d